testsuite/features/support/constants.rb

```diff
@@ -197,12 +197,3 @@
                       'ubuntu1804_minion' => 'x86_64',
                       'ubuntu2004_ssh_minion' => 'x86_64',
                       'ubuntu2004_minion' => 'x86_64' }.freeze
-
-TRADITIONAL_STACK_RPMS = 'spacewalk-client-tools spacewalk-check spacewalk-client-setup '\
-                         'mgr-daemon mgr-osad mgr-cfg-actions'.freeze
-
-OPEN_SCAP_CENTOS_DEPS = 'spacewalk-oscap scap-security-guide'.freeze
-
-OPEN_SCAP_TRAD_DEPS = 'spacewalk-oscap'.freeze
-
-OPEN_SCAP_SALT_DEPS = 'openscap-utils openscap-content scap-security-guide'.freeze
```

Comment:
I don't like the idea to remove these constants.
I prefer to have outside from a method this information, as it's easier to tweak and handle it.

Reply:
I knew you would say that. I still don't understand how having things at different places helps. Having them where they are needed avoid me losing overview.

Said overwise, a constant (or a method) that is used only once does not really make sense to me.

Reply:
That's a lost battle, you will never agree with me in relation to encapsulation and isolation, so... go ahead.